src/Observers/UserSignatureObserver.php

```diff
@@ -2,7 +2,6 @@
 
 namespace Supaapps\Supalara\Observers;
 
-use Illuminate\Support\Facades\DB;
 
 class UserSignatureObserver
 {
```

Comment:
@SirNarsh Is it safe to remove the @todo comment ?

Reply:
Looks good ✅